### DISCUSSION TITLE
part-ii/2021-49/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#402] [Question comments]
Part II 2021 Differential Geometry: Paper 3, Section II, F 

https://questions.tripos.org/part-ii/2021-49/

**Comment by eha1234** [comment]
First few bits are bookwork. 

<details><summary>For the last part</summary>

Suppose not, for contradiction. Then $x \mapsto th(x) + (1-t)h(-x)$ never hits 0. So can define the map $g(x) = \frac{h(x) + h(-x)}{||h(x) + h(-x)||}$ which is smoothly homotopic to f, but has a different number of preimages for some appropriate point in $U$ (picked out to be regular for both maps using Sard). This gives the required contradiction, since $g(x) = g(-x)$ for all $x$ implies that #$g^{-1}(y)$ is even for all $y$. 
</details>

**Reply by ArthurConmy** [reply]
I think there's a typo in $x \mapsto th(x) + (1-t)h(x)x↦th(x)+(1−t)h(x)$ - that's always $h(x)$ no?

**Reply by eha1234** [reply]
Yep sorry! Fixed now to $h(-x)$
